File: app/src/main/java/com/upsage/evosummermobilelab/data/intefaces/AppDatabase.java
package com.upsage.evosummermobilelab.data.intefaces;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.upsage.evosummermobilelab.data.entries.Note;

@Database(entities = {Note.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract NoteDao userDao();
}

File: app/src/main/java/com/upsage/evosummermobilelab/data/intefaces/AppDatabase.kt
package com.upsage.evosummermobilelab.data.intefaces

import androidx.room.Database
import androidx.room.RoomDatabase

import com.upsage.evosummermobilelab.data.Note

@Database(entities = [Note::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): NoteDao
}